src/input/internal.tsx

```diff
@@ -168,6 +176,12 @@ function InternalInput(
     attributes.type = 'text';
   }
 
+  useEffect(() => {
```

Comment:
What was not working with native `autoFocus` attribute, so you had to override its behavior?

Reply:
this is related to the scroll management, native autofocus doesn't maintain user's scroll position. it tries to scroll the element into view